operator/config/webhook/kustomization.yaml

```diff
@@ -3,4 +3,4 @@ resources:
 - service.yaml
 
 configurations:
-- kustomizeconfig.yaml
+- kustomizeconfig.yaml
```

Comment:
@mowies  would not like this : P

Reply:
indeed, and rightfully so :D good catch, fixed